eth_utils/hexadecimal.py

```diff
@@ -4,46 +4,48 @@
 import codecs
 import string
 
+from typing import Any
+
 from .types import (
     is_string,
     is_text,
 )
 
 
-def decode_hex(value):
+def decode_hex(value: str) -> bytes:
     if not is_text(value):
         raise TypeError('Value must be an instance of str')
     return codecs.decode(remove_0x_prefix(value), 'hex')
```

Comment:
Not sure what the best move is here? According to https://github.com/python/typeshed/issues/300 - it seems like mypy won't support accepting strings to `decode()` regardless of the encoding argument. . .

Reply:
It's fine to add a `# type: ignore` to this with a comment linking to the offending issue on github.